11-trt-llm/README.md

```diff
@@ -0,0 +1,75 @@
+## TRT-LLM Builder Truss
+
+### Introduction
+TRT-LLM is high-performance language model inference runtime from Nvidia. To use TRT-LLM on a model, users have to _compile_ a TRT engine from their corresponding PyTorch weights. In this Truss, we provide a config-driven interface to build the engine. We build the engine as part of the server startup and then use Triton Inference Server to serve the engine for high-performance workloads.
```

Comment:
I think this needs more high-level context and maybe some links to some blog posts? I also think it's important to mention that compilation is for a specific GPU and models aren't super portable once compiled.